cmd/monaco/integrationtest/v2/test-resources/integration-bucket/project/bucket.json

```diff
@@ -1,6 +1,5 @@
 {
   "bucketName": "custom_logs",
```

Comment:
We should remove the bucketName from this one too. While deployment will overwrite it, people might look at our test configs as samples and having this field set can be misleading

Reply:
I kept it in with the intent that it should work :(